projects/core/src/config/config-initializer/config-initializer.service.ts

```diff
@@ -19,7 +19,8 @@ export class ConfigInitializerService {
     @Inject(Config) protected config: any,
     @Optional()
     @Inject(CONFIG_INITIALIZER_FORROOT_GUARD)
-    protected initializerGuard
+    protected initializerGuard,
+    @Inject(RootConfig) protected rootConfig: any
```

Comment:
Would you mind adding an unit test that now we also mutate rootConfig on resolved config initializer ?